src/views/ManageList.tsx

```diff
@@ -153,6 +199,24 @@ export function ManageList({ listPath }: Props) {
 						</button>
 					</form>
 					<Toaster />
+					<form onSubmit={(e) => handleInvite(e, listPath)}>
+						<label htmlFor="receipient-email">
```

Comment:
typo: there's an extra `e` in `recipient` here!

Reply:
good catch!